staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util.go

```diff
@@ -608,6 +609,119 @@ func (vs *VSphere) checkDiskAttached(ctx context.Context, nodes []k8stypes.NodeN
 	return nodesToRetry, nil
 }
 
+// BuildMissingVolumeNodeMap builds a map of volumes and nodes which are not known to attach detach controller
```

Comment:
Is this to detect cases where a k8s volume is attached to nodes outside the current k8s cluster?

Reply:
no - I am sorry I chose wrong wording in my original commit message. This case is for detaching volumes from nodes which are still part of k8s cluster but aren't inside attach/detach controller's cache(or ASOW) because that node has no pods with volumes scheduled on it.